components/src/site-footer-condensed/site-footer-condensed.html

```diff
@@ -0,0 +1,79 @@
+<html>
+
+<head lang="en-US">
+	<meta http-equiv="X-UA-Compatible" content="IE=EDGE">
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Site Footer | Component</title>
```

Comment:
Include full component name to match folder structure aka `Site Footer Condensed`